components/ui-api-layer/internal/domain/k8s/resourcequotastatus_resolver_test.go

```diff
@@ -0,0 +1,37 @@
+package k8s
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/kyma-project/kyma/components/ui-api-layer/internal/domain/k8s/automock"
+	"github.com/kyma-project/kyma/components/ui-api-layer/internal/gqlschema"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestResourceQuotaStatusResolver_ResourceQuotaStatus_HappyPath(t *testing.T) {
+	// GIVEN
+	statusChecker := automock.NewResourceQuotaStatusChecker()
+	statusChecker.On("CheckResourceQuotaStatus", fixNamespaceName()).Return(gqlschema.ResourceQuotasStatus{Exceeded: false}, nil)
+
+	resolver := newResourceQuotaStatusResolver(statusChecker)
+
+	// WHEN
+	_, err := resolver.ResourceQuotasStatus(context.Background(), fixNamespaceName())
+	require.NoError(t, err)
```

Comment:
missing // THEN 
missing assertion for returned status from `ResourceQuotasStatus` method